### PR TITLE
Updated localization instructions

### PR DESCRIPTION
**Pull request by akashdabhi03**

### CODE REVIEW

File: LOCALIZE.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ For each `word`, an optional _.png_ image exists. In Swahili, the words _mbu, ki
 
 ### 4. Stories
 Localised stories each have an `id` and reside in `assets/oc-reading/books/xr-[id]/`. Each story has configuration file `book.xml`.
-Presentational apsects of the story are defined on the `book` element. Each `page` contains one or more localised `para`. A `page` can have an optional `picjustify` attribute to specify the page layout.
+Presentational aspects of the story are defined on the `book` element. Each `page` contains one or more localised `para`. A `page` can have an optional `picjustify` attribute to specify the page layout.
 
 In lower level stories, there is a syllable breakdown for each word, delimited by `/`. The title and first page of the _A very tall man_ story in Swahili are represented like this:
```
Comment (mathewnik90) [inline LOCALIZE.md:99]: Can you add the english version of the example?

File: LOCALIZE.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Each `letter` has a unique id and an optional set of `tags`. Possible values for
 ```
 
 ### 2. Phonemes, syllables and words
-The file `assets/oc-literacy-gen/local/LOCALE/wordcomponents.xml` defines the key phonemes, consonant clusters and syllables present in the language. It also contains a curated set of high-frequency and culturally specfic words for learning the language.
+The file `assets/oc-literacy-gen/local/LOCALE/wordcomponents.xml` defines the key phonemes, consonant clusters and syllables present in the language. It also contains a curated set of high-frequency and culturally specific words for learning the language.
 
 Each `phoneme` has a unique `id` with the prefix `is`. In Swahili, the phonemes _a, th_ and _ng'_ are represented like this:
```
Comment (mathewnik90) [inline LOCALIZE.md]: Update to use English examples throughout

File: LOCALIZE.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ For each `word`, an optional _.png_ image exists. In Swahili, the words _mbu, ki
```
Comment (mathewnik90) [inline LOCALIZE.md:93]: Can you update to mention about the process to update the instructions?

File: LOCALIZE.md [hunk 4]
```diff
@@ -121,14 +121,20 @@ Every `para` has a corresponding _.m4a_ recorded audio file and an accompanying
 …
```
Comment (mathewnik90) [inline LOCALIZE.md:121]: Same as above. Use Hindi

File: LOCALIZE.md [hunk 5]
```diff
@@ -121,14 +121,20 @@ Every `para` has a corresponding _.m4a_ recorded audio file and an accompanying
...
-Use [PhraseAnal](https://github.com/XPRIZE/GLEXP-Team-onebillion-PhraseAnal), to assist with generating _.etpa_ files from audio files.
+Each word within a `para` is also individually recorded, along with a version split into syllables for lower-level stories.  
+
+The English transcripts for stories are stored in `assets/transcripts/oc-reading/xr-[id]/english`. Create transcripts for local language and store them in `assets/transcripts/oc-reading/xr-[id]/LOCALE`. Generate audio files in local language and accent from these locale-specific transcripts by using [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html).  
+**Note:** Check for the languages supported by Amazon Polly [here](https://docs.aws.amazon.com/polly/latest/dg/voicelist.html).
+
+After the audio files have been generated, use [Amazon Polly's Speech Marks](https://docs.aws.amazon.com/polly/latest/dg/speechmarkexamples.html), to assist with generating _.txt_ files from transcripts. Create a custom script to generate _.etpa_ files from the text files generated by Amazon Polly.
```
Comment (mathewnik90) [inline LOCALIZE.md]: Can you commit this custom script to GitHub?

How do you generate the _.txt_ file.

File: LOCALIZE.md [hunk 6]
```diff
@@ -121,14 +121,20 @@ Every `para` has a corresponding _.m4a_ recorded audio file and an accompanying
...
 
 
 ### 5. Component audio
 Component localizations consist of a set of _.m4a_ recorded audio files. The file names tend to correspond to the english localization.
 For example in the numeracy component _Add and subtract_, `assets/oc-addsubtract/local/sw/q_sevenbees.m4a` is the Swahili translation of the phrase _"Seven bees"_. In the reading component _Making plurals_, `assets/oc-makingplurals/local/en_GB/mp2_goodtheyreinorder.m4a` is the english recording of _"Good, they are in order"_.
```
Comment (mathewnik90) [inline LOCALIZE.md]: Add english example

File: LOCALIZE.md [hunk 7]
```diff
@@ -121,14 +121,20 @@ Every `para` has a corresponding _.m4a_ recorded audio file and an accompanying
...
-We have provided mappings of all English audio to _.m4a_ filenames. These are xml files inside the `assets/localization` directory.
+We have provided mappings of all English audio to _.m4a_ filenames. These are xml files inside the `assets/localization` directory.  
+
+The English transcripts for all the audio files of a specific module are stored in `assets/transcripts/MODULE/english`. Create transcripts for local language and store them in `assets/transcripts/MODULE/LOCALE`. Generate audio files in local language and accent from these locale-specific transcripts by using [Amazon Polly](https://docs.aws.amazon.com/polly/latest/dg/what-is.html).
```
Comment (mathewnik90) [inline LOCALIZE.md]: Where {LOCALE} is the regional language?
This needs to be more descriptive:
1. Give option to use TTS to generate the audio file
2. Give a brief about using Amazon Polly along with Google Translate to generate audio files. What are the changes needed to support either approaches.

File: LOCALIZE.md (hunk 3 above)
Comment (akashdabhi03) [reply LOCALIZE.md:93]: Already done in section 5 `Component audio`

File: LOCALIZE.md (hunk 1 above)
Comment (akashdabhi03) [reply LOCALIZE.md:99]: Updated

File: LOCALIZE.md (hunk 4 above)
Comment (akashdabhi03) [reply LOCALIZE.md:121]: Updated

File: LOCALIZE.md (hunk 5 above)
Comment (akashdabhi03) [reply LOCALIZE.md]: Will raise a separate PR for the script and specific instructions to run the script.

File: LOCALIZE.md (hunk 2 above)
Comment (akashdabhi03) [reply LOCALIZE.md]: Updated

File: LOCALIZE.md (hunk 6 above)
Comment (akashdabhi03) [reply LOCALIZE.md]: Updated

**Review by mathewnik90 — CHANGES_REQUESTED** [review]

File: LOCALIZE.md (hunk 7 above)
Comment (akashdabhi03) [reply LOCALIZE.md]: Yes LOCALE is the regional language. onebillion team has used it throughout.

TTS is what we are still working on and hence I have not mentioned it.
Regarding the use of Amazon Polly, as I already mentioned, we can create a separate document with instructions on generating audios and etpa files

**Comment by akashdabhi03** [comment]
Closing this pull request due to inactivity

**Pull request closed without merging**